src/components/Exercise.tsx

```diff
@@ -72,6 +74,7 @@ export const Exercise = ({
           canUpdateCurrentStep={canAnswer}
           displaySolution={true}
           detailedSolution={state.solution?.content_html}
+          show_all_feedback={show_all_feedback}
```

Comment:
necessary to show answer feedback when there is no question-level `feedback_html` or `correct_answer_feedback_html`